### PR TITLE
docs: OPTIC-340: Add page on upgrading Label Studio Community

### PR DESCRIPTION
**Pull request by caitlinwheeless**
Adding a new page on how to upgrade Label Studio Community Edition. 

#### Change has impacts in these area(s)

- [X] Community docs
- [ ] Enterprise docs

### CODE REVIEW

**Comment by eurica** [comment]
Could we add a table of contents, like we have on https://deploy-preview-6714--label-studio-docs-new-theme.netlify.app/guide/install

Beyond that, if @jombooth says it's factually correct, I say 🚢 it.

**Review by eurica — APPROVED** [review]

File: docs/source/guide/upgrade_community.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  ```bash
+    python label_studio/manage.py migrate
+  ```
+  1. Collect static files:
+  ```bash
+    python label_studio/manage.py collectstatic
+  ```
+  1. Restart Label Studio:
+  ```bash
+    python label_studio/manage.py runserver
```
Comment (jombooth) [inline docs/source/guide/upgrade_community.md:124]: nit - all these steps seem to be be numbered "1"

File: docs/source/guide/upgrade_community.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+
+## Upgrade after installing from source
+
+If you installed Label Studio [using the source in Github](https://github.com/HumanSignal/label-studio), you can upgrade using the following steps. 
+
+<div class="code-tabs">
+  <div data-name="Using Poetry (recommended)">
+
+  If you'r using Poetry ([see these instructions](install#Install-from-source)), upgrade with the following steps:
```
Comment (jombooth) [inline docs/source/guide/upgrade_community.md]:
```suggestion
  If you're using Poetry ([see these instructions](install#Install-from-source)), upgrade with the following steps:
```

**Review by jombooth — APPROVED** [review]
LGTM, thanks for this! just a couple small nits.

**Pull request merged**